### ISSUE TITLE
Is the code 'model_classifier.train_on_batch*****'  a bug?

### ISSUE THREAD

**Issue by USTClj** [yhenon/keras-frcnn#137]
in train_frcnn.py 

loss_class = model_classifier.train_on_batch([X, X2[:, sel_samples, :]],  # training data
                                                         [Y1[:, sel_samples, :], Y2[:, sel_samples, :]])  # target data
Y2 = np.concatenate([np.array(y_class_regr_label), np.array(y_class_regr_coords)], axis=1)

whereas the output of model_classifier should be y_class_regr_coords.

**Is this a bug?**
